CollarNetworks/protocol-core
# Fix doc issues part 2

Docs:
- adding more contract docs (post-deployment config instructions)
- adding latest mutation testing output
- remove internal review doc (all either resolved or added to known issues list)
- add audit briefing doc
- add some mermaid diagrams

Unrelated:
- fix fork test issue (expected protocol fee was calculated without slippage)
